composer.json

```diff
@@ -18,13 +18,19 @@
         "issues": "https://github.com/stwe/DatatablesBundle/issues"
     },
     "require": {
-        "php": ">=7.1",
-        "symfony/framework-bundle": "^3.0|^4.0",
+        "php": ">=7.2",
+        "symfony/framework-bundle": "^3.4|^4.1",
```

Comment:
Same. Why upgrading Symfony minimum version requirement?

Reply:
BTW if this project can work on 3.4, it should be working on 4.0

Reply:
Support for 4.0 has been dropped: https://symfony.com/roadmap/4.0

I would work on 4.0 in most cases, but not necessarily. I've had cases were 2.8 (LTS) received an imported bugfix for a new PHP release, while a non-LTS 3.x version didn't get that bugfix. 

Supporting an older major Symfony/PHP version (Symfony 3.4, PHP 5.6 when it was still supported) makes sense in most cases, upgrading to a minor release (PHP 7.0 -> PHP 7.1, Symfony 4.0 -> Symfony 4.1) isn't that hard.